CraniobotCourier.m

```diff
@@ -91,6 +91,9 @@ function CraniobotCourier_OpeningFcn(hObject, eventdata, handles, varargin)
     strcat(num2str(100*handles.line/handles.MaxLine),'%')});
 
 % Read in Tool Table
+pathDir='C:\Users\Researcher\Documents\CM\CraniobotCourier-master\CraniobotCourier-master\';
```

Comment:
Is this correct?  Hard coded paths (with specific username even) are generally regarded poorly in most applications.  Not sure about here though. :wink: